src/components/ComponentsListView/ComponentListItem.tsx

```diff
@@ -75,7 +77,7 @@ export const ComponentListItem: React.FC<ComponentListViewPageProps> = ({ compon
           dataListCells={[
             <DataListCell key="name">
               <Flex direction={{ default: 'column' }}>
-                <FlexItem data-testid="component-list-item">
+                <FlexItem data-testid="component-list-item-name">
```

Comment:
@jeff-phillips-18  e2e tests is broken, we need to update this new test id in [Application Page object](https://github.com/openshift/hac-dev/blob/main/integration-tests/support/pageObjects/createApplication-po.ts#L44)